helpers/ginutilext/extension.go

```diff
@@ -0,0 +1,100 @@
+package ginutilext
```

Comment:
Suggest moving this package and this file name

`/helpers/ginutilext/extension.go` &rarr; `/pkg/ginutilext/ginutilext.go`
or 
`/helpers/ginutilext/extension.go` &rarr; `/pkg/ginutilext/writeproblem.go`

"helpers" is unconventional Go naming. A summary of folder names based on conventions, which we've used as a source of truth while learning Go :), is this repo: https://github.com/golang-standards/project-layout